packages/mobile_app/lib/screens/daily/screen.dart

```diff
@@ -34,19 +45,48 @@ class DailyScreen extends StatelessWidget {
           scrollDirection: Axis.vertical,
           children: <Widget>[
             // @poll
-            (dt == currentDay)
-                ? BlocBuilder<UserPollBloc, UserPollState>(
-                    bloc: userPollBloc,
-                    builder: (context, state) {
-                      if (state.enabled == false) return PollSettings();
-                      if (state is UserPollChanged) {
-                        return (userPollBloc.isSimple)
-                            ? PollSimpleWidget(bloc: userPollBloc)
-                            : PollExtendedWidget(bloc: userPollBloc);
-                      }
+            BlocBuilder<UserPollBloc, UserPollState>(
+                bloc: userPollBloc,
+                builder: (context, state) {
+                  switch (state.runtimeType) {
+
+                    /// I am adding debug prints, jsut in case
+
+                    case UserPollLoadingState:
+                      print("User polls are loading.");
+                      return SpinKitPulse(
+                        color: AppColors.accentDark,
+                        size: 32,
+                      );
+
+                    case UserPollLoadingErrorState:
```

Comment:
This state is never used. But exist. Let it be?

Reply:
I guess we would need to add error handling in the future. But I would suggest to remove it anyways. No reason to keep it. Its simple to create it again if needed.